voluptuous/tests/tests.py

```diff
@@ -557,3 +557,31 @@ def fn(arg):
         return "hello"
 
     assert_raises(Invalid, fn, 1)
+
```

Comment:
Add following test cases: 

- [ ]  scale is `None`
- [ ]  precision is `None`
- [ ]  Both are `None`